.vscode/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@RamuniN @tferns Don't think we should commit these kind of config files?

Reply:
@gidsg I think there were some discussions lately about committing vscode configuration for other devs to use/share.  

Not sure the state of that since I don't use vscode, but @RamuniN or @srh-sloan may be able to give more context

Reply:
@gidsg I am creating a confluence document on "**How to attach to a running container & debugging with VsCode**".  We need the settings from `devcontainer.json` in order to do that. I thought this will be the best way to share with the team. I'll share the confluence link once I finish writing.

Reply:
Confluence document is ready here https://digital.dclg.gov.uk/confluence/display/FS/Debugging+container+Flask+apps+with+Vs-Code